dp-terraform/helm/rhacs-terraform/terraform_cluster.sh

```diff
@@ -19,20 +22,14 @@ fi
 ENVIRONMENT=$1
 CLUSTER_NAME=$2
 
-function ensure_bitwarden_session_exists () {
-  # Check if we need to get a new BitWarden CLI Session Key.
-  if [[ -z "$BW_SESSION" ]]; then
-    if bw login --check; then
-      # We don't have a session key but we are logged in, so unlock and store the session.
-      BW_SESSION=$(bw unlock --raw)
-      export BW_SESSION
-    else
-      # We don't have a session key and are not logged in, so log in and store the session.
-      BW_SESSION=$(bw login --raw)
-      export BW_SESSION
-    fi
-  fi
-  bw sync -f
+export AWS_PROFILE="$ENVIRONMENT"
+
+
+# Loads config from the external storage to the environment and applying a prefix to a variable name (if exists).
+load_external_config() {
+    local service="$1"
+    local prefix="${2:-}"
+    eval "$(run_chamber env "$service" | sed -E "s/(^export )(.*)/\1${prefix}\2/")"
```

Comment:
Not sure what guarantees chamber's `env` has, but this might be more robust:
```suggestion
    eval "$(run_chamber env "$service" | sed -E "s/(^export +)(.*)/\1${prefix}\2/")"
```

Reply:
Nice improvement 👍. Done.